### DISCUSSION TITLE
Practice Problem - Design a Restaurant Management System

### DISCUSSION THREAD

**Discussion by kanmaytacker** [kanmaytacker/design-questions#13] [General]
What?
Create an application for management of operations at a restaurant.

Why?
Management system.
Easy to implement end to end and popular for machine coding rounds.

How?
Follow the instructions [here](https://github.com/kanmaytacker/design-questions/blob/master/practice-problems/03-design-restaurant-mgmt-worksheet.md).

**Comment by Mariselvam** [comment]
@kanmaytacker  could you please check the below class diagram for Restaurant management system and let me know your comments..?
![Restaurant-management-system-class-diagram](https://user-images.githubusercontent.com/1678425/213874373-b94e8b3f-ac20-4eb2-bf4f-48b614eb2253.png)
